### DISCUSSION TITLE
Feature Request: Tag/tech tab

### DISCUSSION THREAD

**Discussion by jcardenaslie** [dailydotdev/daily#642] [Feature Requests]
First of all, Im loving the extension, awesome idea. Thanks!

The feature Im requesting is to have a Tab where the tags are grouped.

For example: Currently following python, nodejs, and 7 more.
at one moment I would like to see only my nodejs feed, then my python feed and so on.

:) thanks.

If it already exists please let me know xD

**Comment by idoshamun** [comment]
Hi @jcardenaslie,

Great suggestion and indeed already exists here https://github.com/dailydotdev/daily/discussions/476
